### PR TITLE
Feat: added-necessary-tooltips-for-dokan-settings

### PR DESCRIPTION
**Pull request by MdAsifHossainNadim**
### Added Necessary Tooltips for Dokan Settings

- Updates Fields.vue for get tooltips in frontend.
- Push tooltips from dokan-lite settings for show specific fields tooltip.
- Update admin.less for design.

https://user-images.githubusercontent.com/90011088/139616650-8f8d8552-3782-40ca-9b57-b1bfa893a7e3.png

Related with dokan-pro: <a href='https://github.com/weDevsOfficial/dokan-pro/pull/1562'>https://github.com/weDevsOfficial/dokan-pro/pull/1562/</a>

### CODE REVIEW

File: includes/Admin/Settings.php [hunk 1]
```diff
@@ -443,18 +449,22 @@ public function get_settings_fields() {
...
+                    'tool_tips' => true,
+                    'tips_msg'  => __( 'Enable vendor selling capability upon registration on site', 'dokan-lite' ),
                 ],
                 'disable_product_popup'     => [
                     'name'    => 'disable_product_popup',
                     'label'   => __( 'Disable Product Popup', 'dokan-lite' ),
                     'desc'    => __( 'Disable add new product in popup view', 'dokan-lite' ),
                     'type'    => 'checkbox',
                     'default' => 'off',
+                    'tool_tips' => true,
```
Comment (SadikurSadik) [inline includes/Admin/Settings.php:466]: Indentation problem

File: src/admin/components/Fields.vue [hunk 2]
```diff
@@ -113,10 +122,13 @@
...
             <th scope="row">
                 <label :for="sectionId + '[' + fieldData.name + ']'">{{ fieldData.label }}</label>
+                <span v-if="fieldData.tool_tips">
+                    <i class="dashicons dashicons-editor-help tips" :title="fieldData.tips_msg" v-tooltip="fieldData.tips_msg"></i>
+                </span>
             </th>
             <td>
                 <fieldset>
-                    <label :for="sectionId + '[' + fieldData.name + ']'">
+                     <label :for="sectionId + '[' + fieldData.name + ']'">
```
Comment (SadikurSadik) [inline src/admin/components/Fields.vue:131]: unnecessary extra spaces

**Review by SadikurSadik — CHANGES_REQUESTED** [review]

File: assets/src/less/admin.less [hunk 3]
```diff
@@ -300,6 +300,13 @@
     background: #fff;
 }
 
+.dokan-settings-wrap .metabox-holder form .form-table th {
+  width: 235px;
```
Comment (rajankarmakar) [inline assets/src/less/admin.less:304]: Try to use ( em or rem ) unit instead of px. This would be better for responsiveness.

**Review by SadikurSadik — APPROVED** [review]

**Review by rajankarmakar — APPROVED** [review]

**Review by nurul-umbhiya — APPROVED** [review]

File: includes/Admin/Settings.php [hunk 4]
```diff
@@ -483,6 +490,7 @@ public function get_settings_fields() {
                     'type'    => 'multicheck',
                     'default' => [ 'paypal' => 'paypal' ],
                     'options' => dokan_withdraw_get_methods(),
+                    'tooltip' => __( 'Check to add available payment methods for vendors to withdraw money.', 'dokan' ),
```
Comment (SadikurSadik) [inline includes/Admin/Settings.php:493]: Text domain should be `dokan-lite`

File: includes/Admin/Settings.php [hunk 5]
```diff
@@ -356,6 +356,7 @@ public function get_settings_fields() {
                     'desc'    => __( 'Disable welcome wizard for newly registered vendors', 'dokan-lite' ),
                     'type'    => 'checkbox',
                     'default' => 'off',
+                    'tooltip' => __( 'If checked, vendors will not be prompted through a guided set up process but redirected straight to the vendor dashboard.', 'dokan-lite' ),
```
Comment (SadikurSadik) [inline includes/Admin/Settings.php:359]: `set up` should be `setup`

**Review by SadikurSadik — CHANGES_REQUESTED** [review]

**Review by SadikurSadik — APPROVED** [review]

**Pull request merged**